README.md

```diff
@@ -518,12 +521,101 @@ export const appRouter = createRouter()
 ```
 </details>
 
-## Server-side rendering (SSR / SSG)
+## React-specific helpers (`@trpc/react`)
+
+> _Docs relevant to `@trpc/react`. Follow [Next.js-guide](#nextjs) before doing the below._
+
+### `useInfiniteQuery()`
+
+> - Your procedure needs to accept a `cursor` input of `any` type
+> - For more details read the [react-query docs](https://react-query.tanstack.com/reference/useInfiniteQuery)
+> - Example here is using Prisma - see their docs on [cursor-based pagination](https://www.prisma.io/docs/concepts/components/prisma-client/pagination#cursor-based-pagination)
+
+<details><summary>Example procedure (dummy code)</summary>
+
+```tsx
+import * as trpc from '@trpc/server';
+import { Context } from './[trpc]';
```

Comment:
I don't like importing from a slugified page. Or any page for that matter. But I guess its not wrong for example. I'd just put Context in types.ts or whatever. Not wrong though.

Reply:
I don't love it but i liked it more nor do I like circular reference but I like it more than creating a third file haha